BHoM_UI/Components/Adapter/CreateQuery.cs

```diff
@@ -57,7 +57,7 @@ public class CreateQueryCaller : MethodCaller
         public CreateQueryCaller() : base()
         {
             Type queryType = typeof(BH.oM.DataManipulation.Queries.IQuery);
-            IEnumerable<MethodBase> methods  = BH.Engine.Reflection.Query.BHoMMethodList().Where(x => queryType.IsAssignableFrom(x.ReturnType)).OrderBy(x => x.Name);
+            IEnumerable<MethodBase> methods  = BH.Engine.Reflection.Query.BHoMMethodList(true).Where(x => x.DeclaringType.Name == "Create" && queryType.IsAssignableFrom(x.ReturnType)).OrderBy(x => x.Name);
```

Comment:
This does not build on my machine. `BH.Engine.Reflection.Query.BHoMMethodList(true)` does not take any argument. Guessing you need a engine branch for this as well?

Reply:
Thanks @IsakNaslundBh, your guess is definitely right - I pushed a fixed.